__tests__/app.test.js

```diff
@@ -64,49 +64,67 @@ describe("App API, /api", () => {
             articles.forEach((article) =>
               articleDateArr.push(article.created_at)
             );
-            const sortedDateArr = [...articleDateArr].sort((a, b) => b - a);
-            expect(articleDateArr).toEqual(sortedDateArr);
+            expect(articleDateArr).toEqual([
+              "2020-11-03T09:12:00.000Z",
+              "2020-10-18T01:00:00.000Z",
+              "2020-10-16T05:03:00.000Z",
+              "2020-10-11T11:24:00.000Z",
+              "2020-08-03T13:14:00.000Z",
+              "2020-07-09T20:11:00.000Z",
+              "2020-06-06T09:10:00.000Z",
+              "2020-05-14T04:15:00.000Z",
+              "2020-05-06T01:14:00.000Z",
+              "2020-04-17T01:08:00.000Z",
+              "2020-01-15T22:21:00.000Z",
+              "2020-01-07T14:08:00.000Z",
+            ]);
           });
       });
     });
-    describe.only("GET request with article_id parametric endpoint", () => {
+    describe("GET request for comment by article id", () => {
       it("returns a status of 200 when successfully reached", () => {
-        return request(app).get("/api/articles/1").expect(200);
+        return request(app).get("/api/articles/1/comments").expect(200);
       });
       it("returns a json object with the relevant information", () => {
         return request(app)
-          .get("/api/articles/1")
+          .get("/api/articles/1/comments")
           .expect(200)
           .then(({ body }) => {
-            expect(body).toEqual({
-              article: {
-                article_id: 1,
-                article_img_url:
-                  "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
-                author: "butter_bridge",
-                body: "I find this existence challenging",
-                created_at: "2020-07-09T20:11:00.000Z",
-                title: "Living in the shadow of a great man",
-                topic: "mitch",
-                votes: 100,
-              },
-            });
-          });
-      });
-      it("returns a status of 404 when article id is not found with a message", () => {
-        return request(app)
-          .get("/api/articles/999")
-          .expect(404)
-          .then(({ body }) => {
-            expect(body).toEqual({ message: "Article ID does not exist" });
+            expect(body).toHaveProperty("article_id1 comments");
+            expect(body["article_id1 comments"].length).toBe(11);
+            for (let i = 0; i < 11; i++) {
+              let comment = body["article_id1 comments"][i];
+              expect(comment).toHaveProperty("comment_id");
+              expect(comment).toHaveProperty("votes");
+              expect(comment).toHaveProperty("created_at");
+              expect(comment).toHaveProperty("author");
+              expect(comment).toHaveProperty("body");
+              expect(comment).toHaveProperty("article_id");
```

Comment:
These assertions can be checking the data type instead and the assertion for article_id can instead be exactly what the parametric endpoint is.